### PR TITLE
google-play-music-desktop-player: Remove

### PR DESCRIPTION
**Pull request by klemensn**
`meta.homepage` is unreachable and starting GPMDP eventually just shows
a black screen as described in

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/3942

It also depends on deprecated GNOME2/GTK2.

Google Play Music has been discontinued but this program also supported
YouTube Music, for which also `pkgs.ytmdesktop` exists.

See #39976.
See #115239.
See #115907.

@anna328p @SuprDewd

### CODE REVIEW

**Comment by lf-** [comment]
It's definitely obsolete. I agree you should take it out :)

**Review by lf- — APPROVED** [review]

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -527,6 +527,7 @@ mapAliases ({
   google-gflags = gflags; # Added 2019-07-25
   google-musicmanager = throw "google-musicmanager has been removed because Google Play Music was discontinued"; # Added 2021-03-07
   google-music-scripts = throw "google-music-scripts has been removed because Google Play Music was discontinued"; # Added 2021-03-07
+  google-play-music-desktop-player = throw "GPMDP shows a black screen, upstream homepage is dead, use 'ytmdesktop' instead"; # Added 2022-06-16
```
Comment (AndersonTorres) [inline pkgs/top-level/aliases.nix:530]: You should also effectively remove the `all-packages.nix` call for this package, not only the `default.nix` expression.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/top-level/aliases.nix (hunk 1 above)
Comment (klemensn) [reply pkgs/top-level/aliases.nix:530]: Missed the file during commit, thanks.

**Comment by klemensn** [comment]
Now also with correct GNOME2 deprecation issue number.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Fix the merge conflicts.

**Pull request merged**